src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -1518,6 +1518,9 @@
     <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.SetThreadErrorMode.cs">
       <Link>Common\Interop\Windows\Kernel32\Interop.SetThreadErrorMode.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)Interop\Windows\OleAut32\Interop.SysAllocStringByteLen.cs">
```

Comment:
Ooops, this should be in the block together with other OleAut32 methods. You can clean it up in your next PR.

Reply:
It is under the same, `<ItemGroup Condition="'$(TargetsWindows)' == 'true'">` block which contain, just sorted by filename instead of path.

Reply:
It is under the same, `<ItemGroup Condition="'$(TargetsWindows)' == 'true'">` block which contain other OleAut32 methods, just sorted by filename instead of path.

Reply:
Yes, the convention for project files is to sort by path. It is just a cosmetic issue.

Reply:
Fixed in https://github.com/dotnet/runtime/pull/42058. :)